example/shared_bindings/generated/base_symbols.yaml

```diff
@@ -0,0 +1,26 @@
+format_version: "1.0.0"
+files:
+  ../generated/base_gen.dart:
+    used-config:
+      ffi-native:false
+    symbols:
+      c:@E@BaseEnum:
```

Comment:
are all of the characters ever occurring in here valid yaml keys? Or should we have quotes around them?

Reply:
package:yaml is able to parse these, so I think these are okay.
Anyway, since I've replaced my implementation with package:yaml_edit's it should take care of generating a valid syntax I suppose.